### DISCUSSION TITLE
Problem with lack of audio when using displayport

### DISCUSSION THREAD

**Discussion by DiegoSof2** [prasanthrangan/hyprdots#1016] [General]
The sound only works with the onboard card, it doesn't work when I connect the headphones to the monitor (which works as an amplifier) I've already changed several settings and the best I managed was to get a bit of choppy sound, it seems like it has something to do with Wayland because I've already used it Ubuntu this problem only appeared on wayland

**Comment by katsuja** [comment]
Ubuntu uses gnome under wayland by default

**Comment by prasanthrangan** [comment]
Hey what card are you using? is it nvidia?
I don't have this issue, when i connect headphone to the 3.5mm jack in the monitor it works just fine.

**Comment by DiegoSof2** [comment]
> Hey what card are you using? is it nvidia? I don't have this issue, when i connect headphone to the 3.5mm jack in the monitor it works just fine.

yes gtx 1070, on windows or on a linux not wayland works
